crates/mod-host-assets/src/alloc/default.rs

```diff
@@ -0,0 +1,263 @@
+use std::{
+    mem::{self, ManuallyDrop},
+    ptr::{self, NonNull},
+};
+
+use windows::Win32::System::Memory::{
+    GetProcessHeap, HeapAlloc, HeapFree, HeapReAlloc, HeapSize, HEAP_NONE,
+};
+
+use super::{DlAllocator, DlAllocatorVtable, DlHeapDirection};
+
+const PTR_SIZE: usize = mem::size_of::<*mut u8>();
+const PTR_ALIGN: usize = mem::align_of::<*mut u8>();
+
+pub const DEFAULT_DLALLOC: DlAllocator = DlAllocator {
+    vtable: unsafe { NonNull::new_unchecked(&DEFAULT_DLALLOC_VTABLE as *const _ as _) },
+};
+
+const DEFAULT_DLALLOC_VTABLE: DlAllocatorVtable = DlAllocatorVtable {
```

Comment:
This is ok for now, but any more vtables and we need to start wrapping this up in a proc macro that generates the vtable for us or something similar.